.github/labeler.yml

```diff
@@ -61,7 +61,6 @@ DATA:
   - data/**/*
 SPARK:
   - spark-runtime/**/*
```

Comment:
Technically can't we get rid of most of these now? since everything is spark/**/* now?

Reply:
I thought of it. This file is based on a very old folder structure (not just spark folder problems). So, I thought of doing this file clean-up in the follow-up PR.